recipes/vgraph/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set github_author = "bioinformed" %}
+{% set name = "vgraph" %}
+{% set version = "1.2.0" %}
+# held back until github archives work
+#{% set sha256 = "52a2c85165c75dd6052a07cc31dcf1b10ffde623b4e4423433a35bc726b005f0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  #held back until github archives work
+  #fn: {{ name }}-{{ version }}.tar.gz
+  #url: http://github.com/{{ github_author }}/{{ name }}/archive/v{{ version }}.tar.gz
+  #sha256: {{ sha256 }}
+  git_url: https://github.com/{{ github_author }}/{{ name }}.git
+  git_rev: v{{ version }} 
+  patches:
+    - setup_py.patch
+
+build:
+  number: 0
+  skip: True  # [py<35 and py==36]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
```

Comment:
```diff
+  entry_points:
+    - vgraph = vgraph.vgraph:main
```
That way we let Conda create the the entry point and don't get the `setuptools`' one.